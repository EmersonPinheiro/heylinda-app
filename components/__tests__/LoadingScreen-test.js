import * as React from 'react'
import { render } from '@testing-library/react-native'
import LoadingScreen from '../LoadingScreen'

describe('tests for LoadingScreen component', () => {
  it('renders correctly', () => {
    const tree = render(<LoadingScreen />).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('given Loading screen without props, null should be returned', () => {
    const tree = render(<LoadingScreen />)
    expect(tree).toBeNull()
  })

  // it('given the Loading prop, ActivityIndicator should be returned', () => {
  //   const { getByTestId } = render(<LoadingScreen loading />);
  //   const element = getByTestId('activity-indicator');
  //   expect(element.type).toBe('ActivityIndicator');
  // })
})
