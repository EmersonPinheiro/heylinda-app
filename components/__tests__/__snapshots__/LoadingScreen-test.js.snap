// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests for LoadingScreen component renders correctly 1`] = `null`;
